lib/model_patches.rb

```diff
@@ -57,42 +57,59 @@ def info_requests_hidden_count
     end
   end
 
-  InfoRequest.class_eval do
-    AUSTRALIAN_LAW_USED_READABLE_DATA = {
-      foi: { short: _('FOI'),
-             full: _('Freedom of Information'),
-             act: _('Freedom of Information Act') },
-      gipa: { short: _('GIPA'),
-              full: _("Government Information (Public Access)"),
-              act: _("Government Information (Public Access) Act") },
-      rti: { short: _('RTI'),
-             full: _("Right to Information"),
-             act: _("Right to Information Act") }
-    }
-
-    def australian_law_used
-      if public_body
-        case public_body.jurisdiction
-        when :nsw
-          :gipa
-        when :qld, :tas
-          :rti
-        else
-          :foi
-        end
+  PublicBody.class_eval do
+    def legislation
+      case jurisdiction
+      when :nsw
+        Legislation.find!('gipa')
+      when :qld, :tas
+        Legislation.find!('rti')
       else
-        :foi
+        Legislation.find!('foi')
```

Comment:
This will cause a database hit every time this is called. Is there some caching we can do? the laws don't change all that often